docs/src/adoc/index.adoc

```diff
@@ -684,6 +684,12 @@ handle.createQuery("select * from emotion where emoticon = <sticker>;")
     .list();
 ----
 
+Bindings and definitions are usually separate.  You can define all bindings that don't already
+have template definitions using the `stmt.defineNamedBindings()` or `@DefineNamedBindings` customizers.
+
+By default, to prevent SQL injection, this will bind a boolean indicating only whether the name was bound or not.
```

Comment:
the boolean indicates whether the bound parameter is non-null, not whether it was bound

Reply:
> by default

implies there's a choice in the matter, but — I may be overlooking it — I don't see an enum or implementable interface to have other behavior?